docs/concepts/executor/hub/index.md

```diff
@@ -40,7 +40,6 @@ The Hub architecture looks like the following:
 :hidden:
 
 hub-portal
-create-hub-executor
```

Comment:
What replaces this though? Should we at least say that the creation was already described before?